ngraph/core/include/ngraph/op/random_uniform.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v8
+        {
+            /// \brief Tensor RandomUniform operation.
+            class NGRAPH_API RandomUniform : public Op
+            {
+            public:
+                NGRAPH_RTTI_DECLARATION;
+
+                RandomUniform() = default;
+
+                ///
+                /// \brief      Constructs a RandomUniform operation.
+                ///
+                /// \param      out_shape         Node producing the tensor with output shape.
+                /// \param      min_val           Node producing the tensor with minimum value.
+                /// \param      max_val           Node producing the tensor with maximum value.
+                /// \param      out_type          Output type of the tensor.
+                /// \param      seed              Global seed value.
+                /// \param      seed2             Operational seed value.
```

Comment:
Maybe we can have more human readable arguments (`global_seed`, `op_seed` or something else)?

Reply:
Renamed to global_seed, op_seed.